src/query/RelayQuery.js

```diff
@@ -572,6 +574,42 @@ class RelayQueryRoot extends RelayQueryNode {
     }
     return super.equals(that);
   }
+
+  toJSON(): ConcreteQuery {
```

Comment:
As of @yuzhi's c9fc00541e282e84b6d5e9f58495de477e0753c6, the concrete node no longer needs to be cached, so this can just be `return QueryBuilder.createQuery({...})`